### PR TITLE
Fixes #14246: File lines absent generic method doesn't report anything when the file doesn't exist

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/14246

### CODE REVIEW

File: tree/30_generic_methods/file_ensure_lines_absent.cf [hunk 1]
```diff
@@ -34,15 +34,26 @@ bundle agent file_ensure_lines_absent(file, lines)
...
         classes       => classes_generic_two("${old_class_prefix}", "${class_prefix}");
 
   methods:
+    file_exists::
       "sanitize" usebundle => _classes_sanitize("${class_prefix}");
       "sanitize" usebundle => _classes_sanitize("${old_class_prefix}");
       "report"   usebundle => _log("Removed content from ${file}", "${old_class_prefix}", "${class_prefix}", "${joined}");
+
+    !file_exists::
+      "success"  usebundle => _classes_success("${class_prefix}");
```
Comment (amousset) [inline tree/30_generic_methods/file_ensure_lines_absent.cf]: I'm not sure about the success here (but I get why it can be useful).

Comment (ncharles) [reply tree/30_generic_methods/file_ensure_lines_absent.cf]: it should be _na - closest is success

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**